src/sqlfluff/rules/L006.py

```diff
@@ -189,7 +194,7 @@ def _eval(self, context: RuleContext) -> EvalResultType:
                         LintResult(
                             anchor=after_anchor,
                             description="Missing whitespace after {}".format(
-                                after_anchor.raw[-10:]
+                                after_anchor.raw
```

Comment:
Was this an intentional change? Not sure why it was written the old way, but maybe there was some worry about the string being super long?

Reply:
Yes. Same as above. Not sure why restricted, but not restricted anywhere else. Guess it could lead to very long output but still think clearer.

Reply:
I guess this **is** the lint result message, so it's user facing, unlike most strings in SQLFluff. If we were going to chop it, would probably be good to add "..." when it's too long, so you know something was omitted.

Reply:
Yeah but not chopped in other rules. Probably should create a common function in BaseRule to do a nice chop if we wanted to. But out of scope for this change IMHO.

Reply:
I agree. I was just blathering. 🤣